### DISCUSSION TITLE
Tuner list m3u for specific users

### DISCUSSION THREAD

**Discussion by ruiribeiro2010** [jellyfin/jellyfin#12034] [General]
Jellyfin can add multiple m3u lists, but is it possible to assign only certain users?

example:

list1 - only for user1, user2

list2 - only for user 3, user4

list3 - only for user5. user6

give permission for each user to see only your m3u list.

It was very good, so we could only have 2 or 3 users able to watch streams from a specific list.

thanks

**Comment by mjohl** [comment]
@ruiribeiro2010 Sadly been a feature request for 4 years

https://features.jellyfin.org/posts/461/assign-users-to-tv-tuners

**Reply by ruiribeiro2010** [reply]
Hello, I understand.

but they could implement this feature, I think it is an added value.
Since we can have several m3u lists, we could assign them to different users as necessary.

**Reply by mjohl** [reply]
@ruiribeiro2010 totally agree. I have been thinking of this over the last few days. Been looking for a challenge and might just pull a branch and have a go myself.

**Reply by ruiribeiro2010** [reply]
Hello,
that was great.

If it were not possible to implement directly in jellyfin, it could be implemented as a plugin for anyone who needs it.

**Reply by monforte10** [reply]
Have you been able to make any progress on this topic?

I find this to be a very interesting feature
